CONTRIBUTING.md

```diff
@@ -37,7 +37,22 @@ license.
 
 ### Submitting a contribution, opening an issue.
 
-Please use the github standard interfaces to do so.
+Please use the github standard interfaces to do so. When you submit a
+pull-request, you agree to share your code under `coq-lsp` license.
+
+We have a set of tags to classify pull requests and issues, we try to
+use them as much as possible.
```

Comment:
Is this possible for anybody contributing, or do only elevated privileges have access to labels?

Reply:
Unfortunately I think Github still requires some privileges.

Reply:
Added a few extra wording.